api/api.proto

```diff
@@ -698,6 +698,8 @@ message VolumeSpec {
   bool winshare = 49;
   // Filesystem create options to be honored.
   string fa_create_options = 50;
+  // csi_mount_options user defined mount flags to be honored.
+  string csi_mount_options = 51;
```

Comment:
Since you're updating api.proto, you need to bump the version in this file and add a SDK_CHANGELOG line.